plugins/generator-1.15.2/forge-1.15.2/procedures/entity_is_elytra_flying.java.ftl

```diff
@@ -0,0 +1 @@
+((${input$entity} instanceof PlayerEntity)?((PlayerEntity) ${input$entity}).isElytraFlying():false)
```

Comment:
This method applies to all living entities, please fix accordingly your casts and category of the procedure, same for 1.16.5

Reply:
The player is the only entity able to fly with elytras

Reply:
This does not mean some mod will not extend this functionality for other entities. Always cast to the base class extending a specific method